src/components/UserList/Row.js

```diff
@@ -234,12 +238,14 @@ const styles = StyleSheet.create({
   },
   job: {
     fontSize: 14,
-    marginTop: 25,
-    marginLeft: 25,
+    marginTop: 10,
+    marginLeft: CARD_WIDTH * 0.082,
```

Comment:
'0.082' is used by other lines(242, 243)
We can define a variable and set 0.082.
If this value needs to be changed, We need to change all lines that have '0.082'.